qutebrowser/html/bookmarks.html

```diff
@@ -1,34 +1,95 @@
 {% extends "base.html" %}
 
 {% block style %}
-table { border: 1px solid grey; border-collapse: collapse; width: 100%;}
-th, td { border: 1px solid grey; padding: 0px 5px; }
-th { background: lightgrey; }
+body {
+    background: #fefefe;
+    font-family: sans-serif;
+    margin: 0 auto;
+    max-width: 1280px;
+    padding-left: 20px;
+    padding-right: 20px;
+}
+
+h1 {
+    color: #444;
+    font-weight: normal;
+    margin-bottom: 10px;
+}
+
+a {
+    text-decoration: none;
+    color: #2562dc
+}
+
+a:hover {
+    text-decoration: underline;
+}
+
+.url a {
+    color: #444;
+}
+
+table {
+    border-collapse: collapse;
+    width: 100%;
+}
+
+th {
+    text-align: left;
+}
+
+tbody tr:nth-child(odd) {
+    background-color: #f8f8f8;
+}
+
+.qmarks .name {
+    padding-left: 5px;
+}
+
+.empty {
+    background-color: #f8f8f8;
+    color: #444;
+    display: inline-block;
+    text-align: center;
+    width: 100%;
+}
+
```

Comment:
This duplicates a lot from `history.html`. I suggest creating a `styled.html` instead which does `{% extends "base.html" %}` and adds all the common styling, and then making `history.html` and `bookmarks.html` extend that.

(See [relevant jinja docs](http://jinja.pocoo.org/docs/2.9/templates/#child-template))